app/src/main/java/com/example/myapplication/Calculator.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.myapplication
+
+import java.util.*
+
+object Calculator {
+    fun ExpressionToRpn(expr: String): String {
+        var current = ""
+        val stack = Stack<Char>()
+        var priority: Int
+        for (i in 0 until expr.length) {
+            priority = getP(expr[i])
```

Comment:
можно прямо так делать без объявления отдельной переменной
```suggestion
            val priority = getP(expr[i])
```